design/helm.md

```diff
@@ -0,0 +1,145 @@
+# Helm Addons
+
+To be an addon in a Kurl cluster, they must be contributed into the repo and maintained across new breaking changes and new versions.
+This proposal will give users the flexibility to specify their own addons through a helm chart support in the Kurl installer spec.
+
+
+## Goals
+- Allow any Helm chart to be specified as part of a Kurl spec.
+
+## Non Goals
+
+- Verify authenticity, trustworthyness and "goodness" of helm addons.
```

Comment:
We may not need to verify things directly, but we may need to allow vendor and/or end user to verify things haven't been tampered with. e.g. when a vendor publishes a spec with a certain helm chart, capture fingerprints so the enterprise can later make sure it's the *same* content.

Reply:
Sure, we will definitely need to hash the chart, containers and values to make the airgap bundles. Maybe one new point here is that a hash would always have to include the values (as opposed to a URL as in option 3 of the spec).